docs/environments/find-archived-environment.md

```diff
@@ -0,0 +1,20 @@
+---
+draft: true
+title: Find an Archived an Environment
+intro: Sometimes you may need to inspect which environments have been archived recently. Access the archived environments list to check on all archived environments for the current project.
```

Comment:
1. Same notes on previous comments: the scenario been set is too specific, should be more broader, IMHO, so the readers can identify themselves with the topic even when their use case is not exactly like the one(s) exemplified in the documentation
2. The steps here are stopping after the user access the list. I don't believe we follow these pattern in any other topics written so far. I'd suggest guiding the reader to see the details of an environment. If halfway users are happy enough to see just a list, it's up to them to stop on that step, but that should not be the goal of this topic

```suggestion
intro: Sometimes you may need to access the details of an archived environment or see a list of environments in a project that have been archived. You can access details of all archived environments in a project owned by you.
```

Reply:
@jfoliveira the reason behind the post "cutting short" was because, in the proposed UX design, there's no indication of the user being able to access an archived environment without unarchiving it; I've tried to document the flow that was already described, without adding steps.

In the archived environment list, the only action available on the archived environment card is "unarchive".

Although, as we are documenting this use case early, we can propose changes using this documentation as a means to validate it.

As such, I believe the unarchive flow would be a better fit in a similar flow as to "archive": in the environment details, click `...` and click `Unarchive`

I'll update the PR with this new mindset

Reply:
Updated posts